app/models/reference_repository_denormalizer.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+class ReferenceRepositoryDenormalizer
+  attr_reader :repository
+
+  def initialize(repository)
+    @repository = repository
+  end
+
+  def doi_as_url
+    doi = @repository.re3doi
+    return nil if doi.blank?
+    "https://doi.org/#{doi.downcase}"
```

Comment:
These are always live DOIs regardless of system right? re3data has no staging like we do (handle.stage.datacite.org)

Reply:
Correct